app/android/src/main/java/com/capacitorjs/plugins/app/AppPlugin.java

```diff
@@ -105,6 +105,15 @@ public void getState(PluginCall call) {
         call.resolve(data);
     }
 
+    @PluginMethod
+    public void minimizeApp(PluginCall call) {
```

Comment:
this is not aligned and will cause lint command to fail, can you run `npm run fmt` command? that will fix all lint issues